.github/actions/release_reconfigure/action.yaml

```diff
@@ -21,6 +21,8 @@ runs:
   using: composite
   steps:
     - id: ros_buildfarm_job
+      env:
+        PODMAN_USERNS: keep-id
```

Comment:
Is there an actual effect of setting this in the GitHub actions?

Reply:
It will not do anything right now, no. I set it in these actions specifically because these ones invoke the docker CLI directly, where the others use the scripts and will get `PODMAN_USERNS` from the `job.xml.em` changes.